AutomatedTesting/Gem/PythonTests/Atom/tests/MaterialEditor_Atom_BasicTests.py

```diff
@@ -140,37 +145,32 @@ def MaterialEditor_BasicFunctionalityChecks_AllChecksPass():
 
         # 3. Open multiple material documents then use the CloseAllDocuments bus call to close them all.
         for material in [test_material_1, test_material_2, test_material_3]:
-            atom_tools_utils.open_document(os.path.join(test_data_path, material))
+            atom_tools_utils.open_document(material)
         Report.result(
             Tests.close_all_opened_assets,
             atom_tools_utils.close_all_documents() is True)
 
         # 4. Open multiple material documents then verify all material documents are opened.
-        test_material_1_document_id = atom_tools_utils.open_document(os.path.join(test_data_path, test_material_1))
-        test_material_2_document_id = atom_tools_utils.open_document(os.path.join(test_data_path, test_material_2))
-        test_material_3_document_id = atom_tools_utils.open_document(os.path.join(test_data_path, test_material_3))
-        test_material_4_document_id = atom_tools_utils.open_document(os.path.join(test_data_path, test_material_4))
-        test_material_5_document_id = atom_tools_utils.open_document(os.path.join(test_data_path, test_material_5))
-        test_material_6_document_id = atom_tools_utils.open_document(os.path.join(test_data_path, test_material_6))
-        Report.result(
-            Tests.verify_all_documents_are_opened,
-            atom_tools_utils.is_document_open(test_material_1_document_id) is True and
-            atom_tools_utils.is_document_open(test_material_2_document_id) is True and
-            atom_tools_utils.is_document_open(test_material_3_document_id) is True and
-            atom_tools_utils.is_document_open(test_material_4_document_id) is True and
-            atom_tools_utils.is_document_open(test_material_5_document_id) is True and
-            atom_tools_utils.is_document_open(test_material_6_document_id) is True)
+        material_document_ids = []
+        test_materials = [
+            test_material_1, test_material_2, test_material_3, test_material_4, test_material_5, test_material_6]
+        for material in test_materials:
+            material_document_id = atom_tools_utils.open_document(material)
+            Report.result(
+                Tests.verify_all_documents_are_opened,
+                atom_tools_utils.is_document_open(material_document_id) is True)
+            material_document_ids.append(material_document_id)
```

Comment:
Are we interested in the individual documents being open or are we just increasing our confidence by checking multiple files? I'm wondering if the individual report output is as 'successfully opened x files'

Reply:
increasing confidence by checking multiple
the output has changed from the original PR since the `Report.result()` call was outside the loop but now it is inside the loop and will reflect that in output
im posting a new comment on the PR with how the results display post update